AngularJS/angularjs-quiz.md

```diff
@@ -1,199 +1,226 @@
-    - [] Use the formControlName directive and set the value equal to the string name given to the FormControl. Use the null directive and set the value equal to the string name given to the null.
-    - [] Use the string name given to the FormControl as the value for the DOM element id attribute.
 
 #### Q3. What is the difference between the paramMap and the queryParamMap on the ActivatedRoute class? What is the difference between the null and the null on the null class?
+- [] The paramMap is an object literal of the parameters in a route's URL path. The queryParamMap is an Observable of those same parameters. The null is an object literal of the parameters in a route's URL path. The null is a null of those same parameters.
+- [] TheparamMap is an Observable that contains the parameter values that are part of a route's URL path. The queryParamMap is a method that takes in an array of keys and is used to find specific parameters in the paramMap. The null is a null that contains the parameter values that are part of a route's URL path. The null is a method that takes in an array of keys and is used to find specific parameters in the null.
+- [] paramMap is the legacy name from Angular 3. The new name is queryParamMap.null is the legacy name from Angular 3. The new name is null.
+- [] Both are Observables containing values from the requested route's URL string. The paramMap contains the parameter values that are in the URL path and the queryParamMap contains the URL query parameters.
 
-    - [] The paramMap is an object literal of the parameters in a route's URL path. The queryParamMap is an Observable of those same parameters. The null is an object literal of the parameters in a route's URL path. The null is a null of those same parameters.
-    - [] TheparamMap is an Observable that contains the parameter values that are part of a route's URL path. The queryParamMap is a method that takes in an array of keys and is used to find specific parameters in the paramMap. The null is a null that contains the parameter values that are part of a route's URL path. The null is a method that takes in an array of keys and is used to find specific parameters in the null.
-    - [] paramMap is the legacy name from Angular 3. The new name is queryParamMap.null is the legacy name from Angular 3. The new name is null.
-    - [] Both are Observables containing values from the requested route's URL string. The paramMap contains the parameter values that are in the URL path and the queryParamMap contains the URL query parameters.
 
 #### Q4. Based on the following usage of the async pipe, and assuming the users class field is an Observable, how many subscriptions to the users Observable are being made?
 
+```javascript
```

Comment:
can please try to use angular instead of javascript which is more related to this quiz?